hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieLogFile.java

```diff
@@ -34,6 +34,7 @@
  */
 public class HoodieLogFile {
     public static final String DELTA_EXTENSION = ".log";
+    public static final String ARCHIVE_EXTENSION = ".archive";
```

Comment:
Please dont overload this for representing the archived log file, if thats what you are doing.. This class is reserved for the log files we have with real data. Consider subclass this nicely.